zcash_client_backend/src/data_api.rs

```diff
@@ -236,6 +236,8 @@ pub struct PrunedBlock<'a> {
 pub struct DecryptedTransaction<'a> {
     pub tx: &'a Transaction,
     pub sapling_outputs: &'a Vec<DecryptedOutput>,
+    #[cfg(feature = "transparent-inputs")]
+    pub utxos_spent: &'a Vec<OutPoint>,
```

Comment:
Needs to be added to the changelog. At this point, the entry for `DecryptedTransaction` under "changed" should be split into an "added" and "removed (use instead)".

Reply:
Actually, there's no need to pass the outpoints separately at all; they're already part of the transaction that is passed through, so the only change needed is to always pass the `DecryptedTransaction` to the back end, not just when we successfully decrypted Sapling outputs.